donut/modules/editor/helpers.py

```diff
@@ -125,42 +126,17 @@ def read_file(path):
 
 
 def get_links():
-    '''
-    Get links for all created webpages
-    '''
-    links, root = get_glob()
-    results = {}
-    for filename in links:
-        link = flask.url_for('uploads.display', url=filename)
-        results[filename] = link
+    query = """SELECT title FROM webpage_files"""
+    with flask.g.pymysql_db.cursor() as cursor:
+        cursor.execute(query, [])
+    res = cursor.fetchall()
```

Comment:
I think this line should be inside the `with` statement. Not sure whether it's safe to use the cursor after it's been closed

Reply:
shit u rite